xbmc/cores/VideoPlayer/VideoRenderers/LinuxRendererGL.cpp

```diff
@@ -300,6 +307,22 @@ bool CLinuxRendererGL::Configure(unsigned int width, unsigned int height, unsign
   }
 #endif
 
+  // load 3DLUT
+  if (m_ColorManager->IsEnabled())
+  {
+    if (!m_ColorManager->CheckConfiguration(m_cmsToken, m_iFlags))
+    {
+      CLog::Log(LOGDEBUG, "CMS configuration changed, reload LUT");
+      if (!LoadCLUT())
+        return false;
```

Comment:
Do you think it's ok to disable it here?

A user can go in the settings later, re-enable ColorManager and set the 3dlut file.